magma/array.py

```diff
@@ -272,7 +281,7 @@ def flat_length(cls):
         return cls.N * cls.T.flat_length()
 
     def dynamic_mux_select(self, key):
-        return m.operators.Mux(len(self), self.T)()(*self.ts, key)
+        return Mux(len(self), self.T)()(*self.ts, key)
```

Comment:
I don't see this Mux operator imported above, perhaps we don't have a test code path covering this?

Reply:
Ah, it's defined below.